src/sonic-config-engine/tests/test_cfggen.py

```diff
@@ -68,7 +68,7 @@ def test_additional_json_data(self):
     def test_var_json_data(self):
         argument = '-m "' + self.sample_graph_simple + '" -p "' + self.port_config + '" --var-json VLAN_MEMBER'
         output = self.run_script(argument)
-        self.assertEqual(output.strip(), '{\n    "Vlan1000|Ethernet8": {\n        "tagging_mode": "untagged"\n    }\n}')
+        self.assertEqual(output.strip(), '{\n    "VLAN_MEMBER": {\n        "Vlan1000|Ethernet8": {\n            "tagging_mode": "untagged"\n        }\n    }\n}')
```

Comment:
it looks like you have changed the output format for the var-json output. however, --var-json is used in other places in the buildimage. 
```
lgh@gulv-vm1:/data/sonic/sonic-utilities$ git grep var-json
show/main.py:    neighbor_cmd = 'sonic-cfggen -d --var-json "DEVICE_NEIGHBOR"'
show/main.py:    neighbor_metadata_cmd = 'sonic-cfggen -d --var-json "DEVICE_NEIGHBOR_METADATA"'
show/main.py:    cmd = "sonic-cfggen -d --var-json ACL_RULE"
show/main.py:    cmd = "sonic-cfggen -d --var-json PORT"
```

if the change is necessary, then you need to change places that is using var-json option.

Reply:
These all are parts of show commands only, so I don't think any change is needed here, the change was needed where we are analyzing the output format and that has been changed in test_cfggen.py as it was failing after the change. PORT ACL_RULE I only added it to the show commands to reflect these changes. 
Let me know if you have any questions,

Reply:
what about ```show interface neighbor expected``` cmd, it also uses --var-json option which you have changed the output format.